hal/network/ncp_client/quectel/quectel_ncp_client.cpp

```diff
@@ -1878,7 +1887,7 @@ int QuectelNcpClient::modemPowerOn() {
     if (!modemPowerState()) {
         ncpPowerState(NcpPowerState::TRANSIENT_ON);
 
-        if (ncpId() != PLATFORM_NCP_QUECTEL_BG95_M1 && ncpId() != PLATFORM_NCP_QUECTEL_BG95_MF) {
+        if (!isQuecLPWADevice()) {
```

Comment:
We need to be careful here and make sure that this works correctly.  BG77 might fall into the other category.